### ISSUE TITLE
add support for .org files

### ISSUE THREAD

**Issue by smallstepman** [nschloe/xdoc#53]

**Comment by smallstepman** [comment]
I apologize for not adding any description. 

`.org` files - similar to `.md` or `.rst` -  is a markup file and can be used here on GitHub for building readmes/docs/wiki/else. I opened PR with example `.org` file and some details explaining the issue. 

I really love the idea behind this extension, it shifts the rendering back to the browser which is very easy to use and lightweight solution (especially for inline LaTeX). I'd very appreciate if you could take a stab at it.

**Comment by nschloe** [comment]
Does GitHub render readme.org files?

**Comment by smallstepman** [comment]
yes, it does support most of its features, both as [README](https://github.com/smallstepman/org-file-readme) and as a [directly opened file](https://github.com/smallstepman/org-file-readme/blob/main/README.org)

I encourage you to also check [raw file](https://raw.githubusercontent.com/smallstepman/org-file-readme/main/README.org), `.org` is pretty cool and featureful
